pkgs/top-level/all-packages.nix

```diff
@@ -4424,6 +4424,21 @@ with pkgs;
 
   snooze = callPackage ../tools/system/snooze { };
 
+  cudaToolkitPackages_10_0 = callPackage ./cuda-packages.nix { cudaVersion = "10.0"; };
+  cudaToolkitPackages_10_1 = callPackage ./cuda-packages.nix { cudaVersion = "10.1"; };
+  cudaToolkitPackages_10_2 = callPackage ./cuda-packages.nix { cudaVersion = "10.2"; };
+  cudaToolkitPackages_10 = cudaToolkitPackages_10_2;
+
+  cudaToolkitPackages_11_0 = callPackage ./cuda-packages.nix { cudaVersion = "11.0"; };
+  cudaToolkitPackages_11_1 = callPackage ./cuda-packages.nix { cudaVersion = "11.1"; };
+  cudaToolkitPackages_11_2 = callPackage ./cuda-packages.nix { cudaVersion = "11.2"; };
+  cudaToolkitPackages_11_3 = callPackage ./cuda-packages.nix { cudaVersion = "11.3"; };
+  cudaToolkitPackages_11_4 = callPackage ./cuda-packages.nix { cudaVersion = "11.4"; };
+  cudaToolkitPackages_11_5 = callPackage ./cuda-packages.nix { cudaVersion = "11.5"; };
+  cudaToolkitPackages_11_6 = callPackage ./cuda-packages.nix { cudaVersion = "11.6"; };
+  cudaToolkitPackages = cudaToolkitPackages_11_6;
```

Comment:
What are people's thoughts on naming this simply `cudaPackages`? AFAIU CUDA Toolkit refers to a specific set of packages that does not include CUDA-X packages like cuDNN, cuTENSOR, etc.

Reply:
I feel the same itch! It's a pity that the name is already used and we cannot set up `throw`s to warn people. On the bright side, how likely it is that people have been using `cudaPackages.cudatoolkit_x_y` instead of just `cudatoolkit_x_y`?

Reply:
Apparently extremely unlikely :) Only one instance in all of nixpkgs: https://cs.github.com/?scopeName=All+repos&scope=&q=repo%3Anixos%2Fnixpkgs+cudaPackages.cudatoolkit

Reply:
I want to call it `cudaPackages as well and drop the old one.